android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneginiSDK.kt

```diff
@@ -52,6 +57,22 @@ class OneginiSDK {
         return oneginiClient
     }
 
+    fun getCustomRegistrationActions(): ArrayList<CustomRegistrationAction> {
```

Comment:
Couldn't you just make 
`private var customRegistrationActions = ArrayList<CustomRegistrationAction>()`
public variable? 🤔 Or is it done on purpose?